lib/livebook_web/router.ex

```diff
@@ -40,7 +40,12 @@ defmodule LivebookWeb.Router do
   end
 
   live_session :default,
-    on_mount: [LivebookWeb.AuthHook, LivebookWeb.UserHook, {LivebookWeb.PolicyHook, :private}] do
+    on_mount: [
+      LivebookWeb.AuthHook,
+      LivebookWeb.UserHook,
+      LivebookWeb.HubHook,
```

Comment:
We don't need to run the HubHook on most pages. Only on the Explore/Settings/Home ones. See my comments on HubHook.